src/org/omegat/util/gui/ResourcesUtil.java

```diff
@@ -64,11 +67,28 @@ public static Image getImage(final String resourceName) {
     /**
      * Load icon from classpath.
      *
-     * @param iconName
+     * @param imageName
      *            icon file name
      * @return icon instance
      */
     public static Image getBundledImage(String imageName) {
         return getImage(RESOURCES + imageName);
     }
+
+    /**
+     * Load default colors scheme for LAF theme.
+     *
+     * @param style a style name, should be 'light' or 'dark'
+     * @return Properties object loaded when succeeded, otherwise null
+     */
+    public static Properties getBundleColorProperties(final String style) {
+        String resourcePath = APPROOT + "ColorScheme_" + style + ".properties";
+        Properties properties = new Properties();
+        try (InputStream is = ResourcesUtil.class.getResourceAsStream(resourcePath)) {
+            properties.load(is);
+        } catch (Exception e) {
+            return null;
+        }
```

Comment:
The consumer of this (`UIDesignManager.loadDefaultColors`) blows up via `Objects.requireNonNull` anyway, so I would just make this method throw whatever exceptions it wants and handle them at a higher layer.